hudi-spark-datasource/hudi-spark-common/src/main/java/org/apache/hudi/DataSourceUtils.java

```diff
@@ -293,6 +293,8 @@ public static HiveSyncConfig buildHiveSyncConfig(TypedProperties props, String b
         DataSourceWriteOptions.DEFAULT_HIVE_USE_JDBC_OPT_VAL()));
     hiveSyncConfig.autoCreateDatabase = Boolean.valueOf(props.getString(DataSourceWriteOptions.HIVE_AUTO_CREATE_DATABASE_OPT_KEY(),
         DataSourceWriteOptions.DEFAULT_HIVE_AUTO_CREATE_DATABASE_OPT_KEY()));
+    hiveSyncConfig.ignoreConnectException = Boolean.valueOf(props.getString(DataSourceWriteOptions.HIVE_IGNORE_CONNECT_EXCEPTION_OPT_KEY(),
```

Comment:
Can we extend a larger range from `connect exception` to `sync exception`?

Reply:
@yanghua : are you suggesting, we can make this config cover exceptions in syncHoodieTable() as well ?

Reply:
> @yanghua : are you suggesting, we can make this config cover exceptions in syncHoodieTable() as well ?

Yes, do you have any concerns? In our scenario, we met all hudi spark jobs failed when the hive server is not available.

Reply:
nope. was just clarifying :)

Reply:
@liujinhui1994 : do you have any concerns to make this cover all exception w/ hive sync? I don't see you have addressed this feedback.